### PR TITLE
added documentation and examples to features.rmd

### PR DESCRIPTION
**Pull request by tacaro**
Added documentation and examples to the features.rmd.

### CODE REVIEW

File: vignettes/features.Rmd [hunk 1]
```diff
@@ -481,3 +606,118 @@ mtcars |>
...
+      ) +
+      # set some labels for the base plot
+      labs(
+        # set the shared axis label:
+        x = "Sepal Length"
+      ),
+    # make panel-specific additions
+    add = list(
+      # for Petal.Length panel:
+      ## >>>> add seems to have issues with labs argument
```
Comment (tacaro) [inline vignettes/features.Rmd:647]: @sebkopf The `add` argument doesn't seem to work with `labs()` that are specific to each stacked panel.

File: vignettes/features.Rmd [hunk 2]
```diff
@@ -481,3 +606,118 @@ mtcars |>
...
+        labs(y = "Width of the Sepal"),
+      Petal.Width =
+        geom_line() +
+        coord_cartesian(ylim = c(-3, 5)) +
+        labs(y = "Width of the Petal")
+    )
+  )
+```
+
+# Caution
```
Comment (tacaro) [inline vignettes/features.Rmd:662]: I've added this caution section to help folks avoid accidentally misrepresenting their data. Perhaps this behavior can generate a warning? "Warning: transforming individual stacked panel axes with `add` is not advised."

Comment (sebkopf) [reply vignettes/features.Rmd]: fixed in version >= 0.2.8

File: vignettes/features.Rmd (hunk 1 above)
Comment (sebkopf) [reply vignettes/features.Rmd]: ah good catch @tacaro ! I made a mistake in not checking for `labs(...)` only for `scale_...(name = )` definitions - fixed in version 0.3.0

**Comment by sebkopf** [comment]
a few small changes and fixes (e.g. don't use `library(tidyverse)` in package vignettes, just the packages that are needed since they all need to be bundled with the package) - I think looks great! https://ggstackplot.kopflab.org/

**Pull request merged**